### PR TITLE
Add generate-only option

### PR DESCRIPTION
**Pull request by ali-abrar**
Make it possible to generate the daml test script without running the tests

Also adds some error reporting for parse failures

### CODE REVIEW

File: hs/src/Daml/Cucumber.hs [hunk 1]
```diff
@@ -1,113 +1,144 @@
...
 
-data Opts = Opts
-  { _opts_directory :: FilePath
-  , _opts_featureFile :: Maybe FilePath
-  , _opts_damlSourceDir :: FilePath
-  , _opts_allowMissing :: Bool
-  }
+-- | Check whether a filepath has a particular extension (including ".")
+hasExtension :: String -> FilePath -> Bool
+hasExtension ext path = takeExtension path == ext
```
Comment (madeline-os) [inline hs/src/Daml/Cucumber.hs:33]: this picks up `.daml` the previous implementation implied this was undesirable. does this do the right thing?

Comment (Yasuke) [reply hs/src/Daml/Cucumber.hs:33]: @madeline-os @ali-abrar It does do the right thing, in that it generates the file from only files that are needed, but we should still be disregarding .daml as nothing in there is needed for generation.

**Comment by Yasuke** [comment]
@madeline-os @ali-abrar I pushed up a change adding a verbose flag so that by default we only see the test results.

File: hs/src/Daml/Cucumber.hs (hunk 1 above)
Comment (Yasuke) [reply hs/src/Daml/Cucumber.hs:33]: I guess on a more fundamental note, the file .daml _doesn't_ actually have the extension .daml, which is why I wrote it the way it was. As a dot file is not recognized to have an extension in the way dot files are used.

**Pull request merged**